test/python/test_rticonnextdds_data_access.py

```diff
@@ -393,6 +403,54 @@ def test_shrink_sequence(self, test_output, test_input, test_dictionary):
     assert sample["my_point_array[0].y"] == 20 # Retains value
     assert sample["my_point_array[4].x"] == 14 # Retains value
 
+  def test_too_large_uint64_output(self, test_output):
+    with pytest.raises(rti.Error, match=r".*value of my_uint64 is too large.*") as execinfo:
+      test_output.instance.set_number("my_uint64", 9007199254740992)
+    with pytest.raises(rti.Error, match=r".*value of my_int64 is too large.*") as execinfo:
+      test_output.instance.set_number("my_int64", -9007199254740992)
+
+  def verify_large_integer(self, output, input, number):
+    with pytest.raises(rti.Error, match=r".*value of my_uint64 is too large.*") as execinfo:
+      output.instance.set_number("my_uint64", number)
+    with pytest.raises(rti.Error, match=r".*value of my_int64 is too large.*") as execinfo:
+      output.instance.set_number("my_int64", -number)
+
+    output.instance.set_dictionary({"my_uint64": number, "my_int64":-number})
+    sample = self.send_data(output, input)
+    dictionary = sample.get_dictionary()
+    assert dictionary["my_uint64"] == number
+    assert dictionary["my_int64"] == -number
+    with pytest.raises(rti.Error, match=r".*value of my_uint64 is too large.*") as execinfo:
+      sample.get_number("my_uint64")
+    with pytest.raises(rti.Error, match=r".*value of my_int64 is too large.*") as execinfo:
+      sample.get_number("my_int64")
+
+  def test_large_uint64(self, test_output, test_input):
+    max_int_get = 2**53 # 9007199254740992
```

Comment:
This is the same comment as the one in the lua_binding.1.0 review. I don't understand this discrepancy